grammars/keywords-nb_NO.lark

```diff
@@ -0,0 +1,30 @@
+_PRINT: "skriv" | "print"
+_ASK: "spør" | "ask"
+_ECHO: "ekko" | "echo"
+_FORWARD: "forover" | "forward"
+_TURN: "snu" | "turn"
+right: "høyre" | "right"
+left: "venstre" | "left"
+_IS: "er" | "is"
+_SLEEP: "sov" | "sleep"
+_ADD_LIST: "legg til" | "add"
```

Comment:
This was changed from `legg til` to `legg`

This is to avoid statements like this:
`legg til item til list` which looks silly 😄 
now it will be `legg item til list`

Reply:
It matches well with the `remove` command as well, which I think is a bonus 😊